packages/calypso-e2e/src/types/rest-api-client.types.ts

```diff
@@ -132,9 +132,11 @@ export interface DeleteInvitesResponse {
 	invalid: string[];
 }
 
-export interface NewPostResponse {
+export interface PostResponse {
+	ID: number;
 	URL: string;
 	title: string;
+	status: 'trash' | 'publish' | 'private' | 'draft' | 'future' | 'deleted';
```

Comment:
This is taken from https://developer.wordpress.com/docs/api/1.1/post/sites/%24site/posts/delete/ and simplified.